introduction.md

```diff
@@ -1,2 +1,5 @@
 Hi, my name's Ela and I'm learning git :).
 My favourite Code School path is JavaScript.
+My deadly skills are: 
+* Git
+* Java
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?